migration_toolkit_for_containers/mtc-release-notes.adoc

```diff
@@ -6,12 +6,16 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+The release notes for {mtc-full} ({mtc-short}) describe new features and enhancements, deprecated features, and known issues.
+
 The {mtc-full} ({mtc-short}) enables you to migrate application workloads between {product-title} clusters at the granularity of a namespace.
 
 You can migrate from xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc[{product-title} 3 to {product-version}] and between {product-title} 4 clusters.
 
 {mtc-short} provides a web console and an API, based on Kubernetes custom resources, to help you control the migration and minimize application downtime.
 
+For information on the support policy for {mtc-short}, see link:https://access.redhat.com/support/policy/updates/openshift#app_migration[OpenShift Application and Cluster Migration Solutions], part of the _Red Hat OpenShift Container Platform Life Cycle Policy_
```

Comment:
Thanks, @darshan-nagaraj . I made the change.
In the future, I suggest using the **Add a suggestion** 
![image](https://user-images.githubusercontent.com/39279277/160617799-58ff79b2-69a5-483a-aeee-3477102f3ef4.png)
button, because it makes the diffs in your suggestion much clearer.

Reply:
@stoobie Noted. Thanks for letting me know.